frontend/Themes.jsx

```diff
@@ -122,25 +120,22 @@ export const DataScrollView = styled.SafeAreaView`
 `;
 
 // Styles for Settings Page:
```

Comment:
If this styled component is not only used in the Settings Page this comment should reflect its usage